src/applications/lgy/coe/form/components/SubwayMap.jsx

```diff
@@ -15,7 +15,9 @@ const SubwayMap = () => (
           </li>
           <li>Meet the minimum service requirement based on when you served</li>
         </ul>
-        <a href="/">Eligibility requirements for VA home loan programs</a>
+        <a href="/housing-assistance/home-loans/eligibility">
+          Eligibility requirements for VA home loan programs
+        </a>
       </li>
       <li className="process-step list-two">
```

Comment:
Did you know about the subway web component? https://design.va.gov/storybook/?path=/docs/components-va-process-list--default

Reply:
Yeah, I tried switching to it but in my experience the left padding on the `ol` was different, so I switched back. Might try again when I have more time to mess with it